include/kernel.h

```diff
@@ -4920,7 +4920,7 @@ extern void z_sys_power_save_idle_exit(s32_t ticks);
  * the fatal error handler.
  */
 #define z_except_reason(reason) do { \
-		printk("@ %s:%d:\n", __FILE__,  __LINE__); \
+		__ASSERT_LOC("Exception");\
```

Comment:
What this is going to print is "ASSERTION FAIL [Exception] @ file:line"
But z_except_reason() is called by k_panic/k_oops which are *not* assertions.
I'd rather the output did not include the word "assertion" in it, because it is not.